package.json

```diff
@@ -2,6 +2,10 @@
   "private": true,
   "description": "Material Components Web",
   "license": "Apache-2.0",
+  "keywords": [
```

Comment:
You probably don't need these keywords in our main repo package since it's private and won't ever be published to npm